server/notification-providers/discord.js

```diff
@@ -24,14 +24,17 @@ class Discord extends NotificationProvider {
 
             let url;
 
-            if (monitorJSON["type"] === "port") {
-                url = monitorJSON["hostname"];
-                if (monitorJSON["port"]) {
-                    url += ":" + monitorJSON["port"];
-                }
-
-            } else {
-                url = monitorJSON["url"];
+            switch (monitorJSON["type"]) {
+                case "port":
+                case "ping":
+                    url = monitorJSON["hostname"];
```

Comment:
Doesn't this need to include all of the other none HTTP types such as `dns`?

Reply:
They are covered now